### DISCUSSION TITLE
Has anyone been able to make word timestamp work? And (UserWarning: Failed to launch Triton kernels..)

### DISCUSSION THREAD

**Discussion by AIhasArrived** [openai/whisper#1716] [Q&A]
Hello, 
I tried to search for similar issues and did not find at first glance,
I am tried this command:
`whisper "test2.mp3" --language ... --model large --device cuda -o "test2TranslatedWORDS2" --task transcribe --word_timestamps True --append_punctuations True`

But I am not getting timestamps for words, instead I am getting normal time stamps (for sentences).
__
And next to it I am getting this messages:
C:\Users\userA\AppData\Local\Programs\Python\Python311\Lib\site-packages\whisper\timing.py:42: UserWarning: Failed to launch ```
Triton kernels, likely due to missing CUDA toolkit; falling back to a slower median kernel implementation...
  warnings.warn(
C:\Users\userA\AppData\Local\Programs\Python\Python311\Lib\site-packages\whisper\timing.py:146: UserWarning: Failed to launch Triton kernels, likely due to missing CUDA toolkit; falling back to a slower DTW implementation...
  warnings.warn(
```

**Comment by glangford** [comment]
> But I am not getting timestamps for words, instead I am getting normal time stamps (for sentences).

Check the `.json` output, you should see word timestamps there. The `.srt` file will have normal looking timestamps but refined based on the word timing. To see how accurate the word timing is, you can try `--highlight_words True`.

> ...UserWarning: Failed to launch Triton kernels...

This discussion may be helpful
* https://github.com/openai/whisper/discussions/1283

**Reply by AIhasArrived** [reply]
@glangford  thanks for the answer, I have other questions:
1) Do you know about the DLAS library https://github.com/152334H/DL-Art-School (voice clonage)? How do I make all my GPU be used to make the training FASTER there? Any idea?
2) Do you know of other, better, clone voices githubs?

**Reply by glangford** [reply]
> Do you know about the DLAS library

I am not familiar with it.

**Comment by dgoryeo** [comment]
Does the `UserWarning: Failed to launch Triton kernels..` actually impact the performance or output of the transcription? Thanks.
